packages/dm-core-plugins/src/form/templates/ObjectStorageUncontainedTemplate.tsx

```diff
@@ -28,59 +36,56 @@ export const ObjectStorageUncontainedTemplate = (props: TObjectTemplate) => {
   )
 
   const referenceExists = address !== undefined
-  const canExpand =
-    referenceExists &&
-    (!onOpen ||
-      (uiAttribute?.functionality?.expand ?? config.functionality.expand))
-  const canOpen =
-    referenceExists &&
-    onOpen &&
-    (uiAttribute?.functionality?.open ?? config.functionality.open)
+
+  const { canExpand, canOpenInTab } = InferCanOpenOrExpand(
+    referenceExists,
+    config,
+    uiAttribute
+  )
 
   return (
-    <Fieldset>
-      <Legend>
-        <Typography bold={true}>{getDisplayLabel(attribute)}</Typography>
-        {!config.readOnly && (
-          <SelectReference
-            attributeType={attribute.attributeType}
-            namePath={namePath}
-          />
-        )}
-        {attribute.optional && address && !config.readOnly && (
-          <RemoveObject address={address} namePath={namePath} />
-        )}
-        {canOpen && (
-          <OpenObjectButton
-            viewId={namePath}
-            viewConfig={
-              uiAttribute?.openViewConfig
-                ? uiAttribute?.openViewConfig
-                : {
-                    type: 'ReferenceViewConfig',
-                    scope: namePath,
-                    recipe: uiAttribute?.uiRecipe,
-                  }
-            }
-            idReference={address}
-          />
-        )}
-      </Legend>
-      {canExpand && (
-        <ViewCreator
-          idReference={address}
-          onOpen={onOpen}
-          viewConfig={
-            uiAttribute?.expandViewConfig
-              ? uiAttribute?.expandViewConfig
-              : {
-                  type: 'ReferenceViewConfig',
-                  recipe: uiAttribute?.uiRecipe,
-                }
+    <FormObjectBorder>
```

Comment:
Suggestion: Rename to something like this (similar to the eds Dialog).

```
<Template> (FormObjectBorder)
    <Template.Header> (ObjectLegendWrapper)
    <Template.Title>  (ObjectLegendHeader)
    <Template.Actions> (ObjectLegendActionsWrapper)
         ...
    </Template.Actions>
    </Template.Header>
  <Template.Content> (FormExpandedViewWrapper)
   ...   
   </Template.Content>
  </Template>
```

Reply:
https://eds.equinor.com/0b0c666ab/p/059c0e-dialog/b/1592fb

Reply:
Good suggestion. Done.